pkg/jar/parse.go

```diff
@@ -418,7 +421,7 @@ func searchBySHA1(c conf, data []byte) (properties, error) {
 	}
 
 	if len(res.Response.Docs) == 0 {
-		return properties{}, xerrors.Errorf("no artifact found: %s", digest)
+		return properties{}, xerrors.Errorf("digest %s: %w", digest, ArtifactNotFoundErr)
```

Comment:
We should also return an error if searchBySHA1 returns an error other than ArtifactNotFoundErr.